app/src/main/java/com/spendster/presentation/homeScreen/PageFragment.java

```diff
@@ -0,0 +1,33 @@
+package com.spendster.presentation.homeScreen;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class PageFragment extends Fragment {
```

Comment:
@SimonMaha this name is too general. Already contain some specific logic. The better naming would be DashboardPageFragment